examples/events/snowplow-debugger/recommended-item.json

```diff
@@ -54,7 +58,8 @@
                   "type": "object",
                   "properties": {
                     "code": {
-                      "type": "string"
+                      "type": "string",
```

Comment:
I have rarely seen finalAdjustments being set in price data. Doesn't look like it's marked as required, we can remove this